frontend/simple/js/events.js

```diff
@@ -44,6 +65,17 @@ export class GroupContract extends Events.HashableGroup {
           await backend.subscribe(identityContractId)
           await store.dispatch('syncContractWithServer', identityContractId)
         }
+      },
+      async  HashableGroupVoteForOrAgainstProposal ({state}, data) {
```

Comment:
Can we make it so that the async actions here are consistent with the Vuejs actions?

i.e. could you convert this to:

```js
async  HashableGroupVoteForOrAgainstProposal ({state}, {data}) {
```